src/Microsoft.FeatureManagement/FeatureManager.cs

```diff
@@ -172,124 +182,111 @@ private async Task<bool> IsEnabledAsync<TContext>(string feature, TContext appCo
 
             bool enabled;
 
-            FeatureDefinition featureDefinition = await _featureDefinitionProvider.GetFeatureDefinitionAsync(feature).ConfigureAwait(false);
-
-            if (featureDefinition != null)
+            //
+            // Treat a null, empty, or status disabled feature as disabled
+            if (featureDefinition == null || 
```

Comment:
`featureDefinition == null` we don't call this with null featureDefinition do we?